### PR TITLE
Update README to include help on using extensions

### PR DESCRIPTION
**Pull request by calebkiage**
## Overview

Adds docs on using the extension command

## Testing Instructions
N/A

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoftgraph/msgraph-cli/pull/193)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,29 @@ You can use the `--query` parameter and the [JMESPath](http://jmespath.org/) que
 $ mgc users user list --query "[?userType=='Member'].{name: displayName, email: mail}" --output table
 ```
 
+#### Extensions
```
Comment (jobala) [inline README.md:60]: It is worth mentoning that they -- our customers -- can get beta commands through this functionality

Comment (calebkiage) [reply README.md:60]: Done :-)

**Review by jobala — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -57,6 +57,30 @@ You can use the `--query` parameter and the [JMESPath](http://jmespath.org/) que
...
+You can manage extensions using the `extension` command.
```
Comment (MIchaelMainer) [inline README.md:61]: It would be helpful to inform the reader what is an extension.

Comment (calebkiage) [reply README.md:61]: I mention on L62 that beta commands are shipped as extensions.

**Review by samwelkanda — APPROVED** [review]

**Pull request merged**